### PR TITLE
feat: Expose Polling Status

### PR DESCRIPTION
**Pull request by nico-ulbricht**
Since `Consumer.stopped` is a private member of the Consumer, it is right now not possible to check if it's running or not from the outside.

In order to see from the outside if the polling has correctly started, I propose exposing the boolean through a getter `isRunning`.

### CODE REVIEW

File: src/consumer.ts [hunk 1]
```diff
@@ -119,6 +119,10 @@ export class Consumer extends EventEmitter {
     autoBind(this);
   }
 
+  public get isRunning(): boolean {
```
Comment (nspragg) [inline src/consumer.ts]: Please update the docs for `isRunning`

**Comment by nspragg** [comment]
@nico-ulbricht - Can update the docs for `isRunning`?

**Comment by nico-ulbricht** [comment]
@nspragg Done, added it to the README

**Comment by nspragg** [comment]
@nico-ulbricht - thanks for your contribution.

**Pull request merged**